benchmarking/blackbox_repository/blackbox_surrogate.py

```diff
@@ -106,15 +95,38 @@ def fit_surrogate(self, surrogate=KNeighborsRegressor(n_neighbors=1)) -> Blackbo
         if len(numeric) > 0:
             features_union.append(('numeric', make_pipeline(Columns(names=numeric), StandardScaler())))
 
-        self.surrogate_pipeline = Pipeline([
+        return Pipeline([
             ("features", FeatureUnion(features_union)),
-            ('model', surrogate)
+            ('standard scaler', StandardScaler(with_mean=False)),
+            ('model', model)
         ])
 
-        self.surrogate_pipeline.fit(
-            X=self.X,
-            y=self.y
+    def fit_surrogate(self, X, y, surrogate=KNeighborsRegressor(n_neighbors=1), max_samples: Optional[int] = None) -> Blackbox:
+        """
+        Fits a surrogate model to a blackbox.
+        :param surrogate: fits the model and apply the model transformation when evaluating a
+        blackbox configuration. Possible example: KNeighborsRegressor(n_neighbors=1), MLPRegressor() or any estimator
+        obeying Scikit-learn API.
+        """
+        self.surrogate = surrogate
```

Comment:
I'd expect something like if surrogate is None, use self.surrogate ?

Reply:
That would not work with how the method is called.